mixmatch/services.py

```diff
@@ -108,3 +116,13 @@ def _is_reverse(order):
         return True
     else:
         raise ValueError
+
+
+def _remove_details(volumes):
+    """Delete key, value pairs if key is not in keys"""
+    keys = ['id', 'links', 'name']
+    for i in range(len(volumes)):
+        for key, value in volumes[i].items():
```

Comment:
Use list comprehension. [Source](http://stackoverflow.com/questions/3420122/filter-dict-to-contain-only-certain-keys)

```python
>>> a = { 'a': 0, 'b': 1, 'c': 2, 'd': 3 }
>>> { key: a[key] for key in ['a', 'b'] }
{'a': 0, 'b': 1}
```